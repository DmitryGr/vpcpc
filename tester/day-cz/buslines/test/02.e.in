30
5 6
26 10
14 13
26 22
30 26
18 6
19 4
26 19
25 16
4 21
18 20
29 25
17 28
15 13
12 26
29 26
26 17
3 2
11 26
23 26
16 13
1 26
18 14
26 2
8 26
26 7
24 21
11 9
17 27
