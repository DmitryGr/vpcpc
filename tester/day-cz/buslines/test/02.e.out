29
57
29
83
29
57
29
29
29
29
57
29
172
129
29
183
84
109
107
29
57
29
29
29
197
379
29
29
209
29
